1832.check-if-the-sentence-is-pangram.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * @param {string} sentence
+ * @return {boolean}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/check-if-the-sentence-is-pangram/
+*/
+
+ var checkIfPangram = function(sentence) {
+    const alphabetTable = {};
```

Comment:
```suggestion
    const alphabetMap = new Map();
```

Reply:
I updated the name and the initialization of alphabetTable as you described above.